iml-request-retry/examples/server.py

```diff
@@ -0,0 +1,43 @@
+import time
+import flask
+from flask import Flask, request, jsonify
+
+app = flask.Flask(__name__)
+
+# single threaded server only
+counter = {}
+
+
+@app.route('/echo')
+def echo():
+    return flask.request.data
+
+
+@app.route('/timeout')
+def timeout():
+    time.sleep(flask.request.args.get('sleep', 1))
+    return 'ok'
+
+
+@app.route('/fail')
+def fail():
+    raise RuntimeError('failed')
+
+
+@app.route('/fail5')
+def fail5():
+    # method, that fails, when counter2 is odd
```

Comment:
This looks like a misleading comment, actual code fails when `% 5 != 0`

Reply:
yep